code/modules/modular_computers/file_system/programs/app_presets.dm

```diff
@@ -0,0 +1,146 @@
+/datum/modular_computer_app_presets
```

Comment:
File is indented with spaces, as opposed to tabs. Should be converted to use tabs.